### PR TITLE
Clarify comment in scroll-snap-align css example

### PR DESCRIPTION
**Pull request by inferrinizzard**
### Description

Rewords a comment on the `scroll-snap-align` page css example to be more clear

### Motivation

The previous text was unclear and grammatically incorrect

### CODE REVIEW

File: files/en-us/web/css/scroll-snap-align/index.md [hunk 1]
```diff
@@ -16,7 +16,8 @@ The `scroll-snap-align` property specifies the box's snap position as an alignme
 ```css
 /* Keyword values */
 scroll-snap-align: none;
-scroll-snap-align: start end; /* when two values set first is block, second inline */
+/* when two values are set, the first value controls the block axis and the second value controls the inline axis */
```
Comment (dipikabh) [inline files/en-us/web/css/scroll-snap-align/index.md:19]: We usually document the meaning of specifying one or two values in prose. For an example, see [`padding-block`](https://developer.mozilla.org/en-US/docs/Web/CSS/padding-block#syntax).

In this case, I would recommend redoing `/* Keyword values */` as `/* Single keyword value */` and adding another section for `/* Two keyword values */`.

**Review by dipikabh — CHANGES_REQUESTED** [review]
Hi @inferrinizzard, thanks for your contribution and for helping us improve the docs.

Please take a look at the feedback and let me know if you'd be willing to make the requested changes or if you have any questions. Thanks!

**Review by dipikabh — CHANGES_REQUESTED** [review]
Thanks for the updates, @inferrinizzard.

Let's move out the verbose explanations from comments into prose.

**Review by dipikabh — APPROVED** [review]
I moved the sentence to the "Values" section as per our usual format. Thanks for the updates 👍

**Comment by dipikabh** [comment]
Congratulations, @inferrinizzard, on landing your first PR in MDN! Welcome to the community 🎉

**Comment by inferrinizzard** [comment]
Thanks for the feedback, @dipikabh !

**Pull request merged**